src/configure-webpack/index.js

```diff
@@ -83,6 +83,16 @@ const buildSharedWebpackConfig = (saguiConfig) => {
       })
     ],
 
+    // In wepback 2 builtin node.js modules take precedence over the normal modules
+    // So, if a user creates a `constants` module and tries to require it
+    // with `import 'constants'`, the node.js module will be imported instead
+    //
+    // This configuration disables this behavior
+    // More information: https://github.com/webpack/webpack/issues/4666#issuecomment-292700060
+    node: {
+      constants: false
+    },
+
```

Comment:
Thanks for reviewing ❤️

The problem is more related to trying to suppress long relative paths. Such as `require('../../../../constants')`, Sagui currently supports the root `src/` directory so that you can just `require('constants')` anywhere in the directory tree. 

If we want to go towards that direction, we should instead force users to always have relative paths for all of their imports.

Btw, this issue manifested in the [Day One source code](https://github.com/macaco-maluco/day-one/blob/master/src/selectors/visible-universe/noise-matrix.js#L3).

Given I don't think someone would like to require the actual Node.js module `constants` in a front-end project, this proposed change is not likely to be a problem.

We could have this merged and release as part of the RC releases and maybe change it latter before the final release if we stumble on any issue.

Wdyt?